apps/ui/src/components/SpaceSettingsAlerts.vue

```diff
@@ -43,14 +43,25 @@ const hasVotingStrategiesAlerts = computed(
       unsupportedProOnlyNetworks.value.length > 0)
 );
 
+const isRelayerBalanceLow = computed(() =>
+  alerts.value.has('IS_RELAYER_BALANCE_LOW')
+);
+
```

Comment:
This new computed doesn't match naming of other flags (used to be `hasABCAlerts`) so probably should be `hasRelayerBalanceAlerts` here.

btw. not sure why it's `alerts` instead of `alert` in those flags (`hasAnyAlerts` makes sense, but individual flags I feel like should be `hasRelayerBalanceAlert`, `hasWhitelabelAlert`, `hasAuthenticatorAlert`, `hasStrategiesAlert`). What do you think? If you agree no need to unify in this PR, let's create issue for this.